test/sequential/feeLock.API.config.test.ts

```diff
@@ -142,7 +146,7 @@ test("gasless- Changing feeLock config parameter on the fly is works robustly",
   let updateMetadataEvent: any;
   testUser1.addAsset(MGA_ASSET_ID);
 
-  const feeLockAmount = new BN(
+  const feeLockAmount = stringToBN(
```

Comment:
This is the same question about `getFeeLockMetadata`

Reply:
yup, will fix this,